### PR TITLE
Template | Fix #2 | Related with #3

### LINKED ISSUE #2: Create a template slide
To ensure that the slides have a minimum standardisation, create a template that will serve as the basis for all future slides:

The template must include

- [ ] Standard names
- [ ] footnote: GH link, material link, RC team homepage, RC ticket link (?)
- [ ] Set theme
- [ ] Standard page title
- [ ] If necessary: Custom style

### PR DESCRIPTION
**Pull request by patricia-ternes**
# Create a Template for standard training courses

This PR Includes:

- New Title slide (`_title.qmd`): individual file with course code, course name, standard training banner
- New Team slide(`_team.qmd`): individual file with team related info
- New End file (`_end.qmd`): last slide with thanks and some useful links
- New Variables file (`_variables.yml`): file to define all values >> which allows a reliable and fast update of any data in all presentations
- New Template Presentation file (`template.qmd`): file that organise all above to generate a standard presentation
- Update `README.md` file: include explanation about the standard presentation (all above)
- Update `index.qmd` file: update name, include list of possible presentations.

### CODE REVIEW

File: _end.qmd [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Thank you {.smaller}
+
+If you have any questions or would like to learn more about research computing,
```
Comment (Sparrow0hawk) [inline _end.qmd:3]:
```suggestion
If you have any questions or would like to learn more about Research Computing,
```

File: _end.qmd [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+please do not hesitate to get in touch with us.
+
+\
+
+### We are always here to assist you!
+
+- Contact via IT Servicedesk: {{< var rc.servicedesk >}}
```
Comment (Sparrow0hawk) [inline _end.qmd:10]:
```suggestion
- Contact via IT Service Desk: {{< var rc.servicedesk >}}
```

File: _team.qmd [hunk 3]
```diff
@@ -0,0 +1,14 @@
+## Research Computing support team {.smaller}
```
Comment (Sparrow0hawk) [inline _team.qmd:1]:
```suggestion
## Research Computing Support Team {.smaller}
```

I don't know if you could drop Support and just have it be `Research Computing Team`?

Comment (patricia-ternes) [reply _team.qmd:1]:
```suggestion
## Research Computing Team {.smaller}
```

**Pull request merged**